licensetest/license_test.go

```diff
@@ -29,6 +30,9 @@ func TestLicense(t *testing.T) {
 			}
 			return nil
 		}
+		if strings.Contains(path, "testbin") {
```

Comment:
To exclude certain files from the license check, they can be included in the skip map like this
```
"../testbin": true, // Generated files
```